

// describe("[Auth]: Test Login Service", () => {
    
    
//     it("Should succeed Logging In", async () => {
//         throw new Error("Should succeed Logging In is not implemented");
//     });

//     it("Should succeed non existing account", async () => {
//         throw new Error("Should succeed non existing account is not implemented");
//     });

//     it("Should fail, remote call error", async () => {
//         throw new Error("Should fail, remote call error is not implemented");
//     });

//     it("Should fail, error creating new local account", async () => {
//         throw new Error("Should fail, error creating new local account is not implemented");
//     });

//     it("Should fail, error persisting new token", async () => {
//         throw new Error("Should fail, error persisting new token is not implemented");
//     });

//     it("Should fail, unable to publish event to Event Bus", async () => {
//         throw new Error("Should fail, unable to publish event to Event Bus is not implemented");
//     });

//     it("Should fail, no unit of work provided", async () => {
//         throw new Error("Should fail, no unit of work provided is not implemented");
//     });

//     it("Should fail, no principal provided", async () => {
//         throw new Error("Should fail, no principal provided is not implemented");
//     });
// });

// describe("[Auth]: Test Refresh Service", () => {
//     it("Should succeed Refreshing Session", async () => {
//         throw new Error("Should succeed Refreshing Session is not implemented");
//     });

//     it("Should fail, no old session found", async () => {
//         throw new Error("Should fail, no old session found is not implemented");
//     });

//     it("Should fail, remote call error", async () => {
//         throw new Error("Should fail, remote call error is not implemented");
//     });

//     it("Should succeed, auth session not found in server", async () => {
//         throw new Error("Should succeed, auth session not found in server is not implemented");
//     });

//     it("Should succeed, auth session not authorized", async () => {
//         throw new Error("Should succeed, auth session not authorized is not implemented");
//     });

//     it("Should fail, error persisting new token", async () => {
//         throw new Error("Should fail, error persisting new token is not implemented");
//     });

//     it("Should fail, error publishing event to EventBus", async () => {
//         throw new Error("Should fail, error publishing event to EventBus is not implemented");
//     });

//     it("Should fail, no unit of work provided", async () => {
//         throw new Error("Should fail, no unit of work provided is not implemented");
//     });

//     it("Should fail, no account id provided", async () => {
//         throw new Error("Should fail, no account id provided is not implemented");
//     });
// });

// describe("[Auth]: Test Logout Service", () => {
    
//     it("Should succeed logging out", async () => {
//         throw new Error("Should succeed logging out is not implemented");
//     });

//     it("Should fail, auth repository error", async () => {
//         throw new Error("Should fail, auth repository error is not implemented");
//     });

//     it("Should fail, remote call error", async () => {
//         throw new Error("Should fail, remote call error is not implemented");
//     });

//     it("Should fail, failed to publish event into EventBus", async () => {
//         throw new Error("Should fail, failed to publish event into EventBus is not implemented");
//     });
// });

// describe("[Auth]: Test Get Active Session Service", () => {

//     it("Should succeed getting active session", async () => {
//         throw new Error("Should succeed getting active session is not implemented");
//     });

//     it("Should fail, auth repository error", async () => {
//         throw new Error("Should fail, auth repository error is not implemented");
//     });

//     it("Should fail, not unit of work provided", async () => {
//         throw new Error("Should fail, not unit of work provided is not implemented");
//     });
// });